pkg/component/operator/web/v0/crawl_website.go

```diff
@@ -177,6 +182,12 @@ func (e *execution) CrawlWebsite(input *structpb.Struct) (*structpb.Struct, erro
 		if len(output.Pages) < inputStruct.MaxK {
 			output.Pages = append(output.Pages, page)
 
+			// Signal that we've added a new page
+			select {
+			case pageUpdateCh <- struct{}{}:
+			default:
```

Comment:
Why do you need a select with default here instead of sending a value to the channel directly?